test/parallel_api/numeric/numeric.ops/reduce_by_segment.pass.cpp

```diff
@@ -119,6 +119,10 @@ void test_with_buffers()
 void test_with_usm()
 {
     sycl::queue q;
+
+    if(!TestUtils::has_aspect(q.get_device(), sycl::aspect::usm_shared_allocations))
```

Comment:
I think that this function should should become boolean. And when we return from this function, we should return as in skip test vase from main.

Reply:
Let's continue discussion on the same issue in this thread: https://github.com/oneapi-src/oneDPL/pull/420#discussion_r769307204